src/components/StackedChart/index.js

```diff
@@ -142,32 +138,32 @@ const StackedChart = ({ title, type, data, isCurrency, showTimeFilter, maxHeight
       <ResponsiveContainer maxHeight={maxHeight} maxWith={maxWith} minHeight={minHeight}>
         {type === 'AREA' ? (
           <AreaChart
-            className="basic-chart"
+            className={'basic-chart'}
```

Comment:
```suggestion
            className={'basic-chart'}
```
Why string prop changed to ={''} way?

Reply:
We have both syntaxes across the app so I was changing them to be `={}` where I found them, so it's consistent across the code. Maybe we can add a lint rule for this if it makes sense.